src/pages/registration/description_tab/projects_field/projectHelpers.ts

```diff
@@ -62,3 +62,5 @@ export const projectContributorToCristinPerson = (
         employments: [],
       }
     : null;
+
+export const isRekProject = (project?: CristinProject) => project?.created.sourceShortName === 'REK';
```

Comment:
Har ikke fått endelig avklart at dette er den beste måten å avgjøre om det er et REK-prosjekt, men det virker slik for meg

Reply:
Bekreftet fra Daniel S